docs/pages/Layout/+Page.mdx

```diff
@@ -138,12 +140,15 @@ To fetch data for your layouts see:
 
 You can access <Link href="/pageContext">the `pageContext` object</Link> by using <Link href="/usePageContext">`usePageContext()`</Link>.
 
-```jsx
-// /pages/+Layout.jsx
+```tsx
+// /pages/+Layout.tsx
+
+export { Layout }
```

Comment:
Love it :heart_eyes: (really nice to make it consistent with the rest of the page's code blocks!)